src/client-entry.js

```diff
@@ -135,12 +135,22 @@ router.onReady(() => {
 		// get newly activated components
 		const matched = router.getMatchedComponents(to);
 		const prevMatched = router.getMatchedComponents(from);
-		const activated = _dropWhile(matched, (c, i) => prevMatched[i] === c);
+		const areRoutesTheSame = JSON.stringify(to?.matched?.[0]?.path) === JSON.stringify(from?.matched?.[0]?.path);
+		const areParamsTheSame = JSON.stringify(to?.params) === JSON.stringify(from?.params);
```

Comment:
Do we need to check if params are not in the same order?

Reply:
vue router returns them in the order they are defined in the route. So for a fictional route of 
`/lend-by-category/:category/:test` they always come back as {category: "foo", test: "bar"}

Reply:
Sounds good.